.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
See [this page](https://docs.travis-ci.com/user/languages/ruby). It describes a `bundler_args` option which sounds like exactly what we need.